### ISSUE TITLE
this project is dead?

### ISSUE THREAD

**Issue by git-xyz** [SimulatedGREG/electron-vue#1110]
#### Found an issue or bug with electron-vue? Tell me all about it!
Questions regarding how to use `electron` or `vue` are likely to be closed as they are not direct issues with this boilerplate. Please seek solutions from official documentation or their respective communities. 

##### Describe the issue / bug.
\#

##### How can I reproduce this problem?
\#

##### If visual, provide a screenshot.
\#

##### Tell me about your development environment.
  * Node version:
  * NPM version:
  * vue-cli version: (if necessary)
  * Operating System:

**Comment by Deluze** [comment]
Yes. there are blatant errors in the build script and this boilerplate doesn't really work "out of the box", the maintainers are not bothered fixing it.

I made my own Vue 3 + Electron template with ViteJS and Electron Builder to learn more about ViteJS and Electron. Project can be found here: [electron-vue-template](https://github.com/Deluze/electron-vue-template). I currently use this in my personal projects.

**Comment by jooy2** [comment]
I've split this into a new project with regards to it: https://github.com/jooy2/vutron

- Supports Vue 3, Electron 19~, NodeJS 14, 16~
- Supports the latest development environments such as Webpack 5, Babel 7, ESLint 8, etc.
- Faster compilation speed, various build problems fixed

I don't know if `electron-vue` will continue to be maintained, but I personally started working on new projects because I used this template often.

Please take a look around if you are interested.
